util/util.go

```diff
@@ -241,7 +248,7 @@ func ReadFile(fs billy.Basic, name string) ([]byte, error) {
 		return nil, err
 	}
 
-	defer f.Close()
+	defer func() { _ = f.Close() }()
```

Comment:
Not sure why we need this change if we are not handling nor logging the error.

Reply:
Was just fixing an goland nag, i'll remove.